com.unity.render-pipelines.universal/Editor/Converter/Converters/RenderSettingsConverter.cs

```diff
@@ -0,0 +1,301 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor.Search;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+using ShadowQuality = UnityEngine.ShadowQuality;
+using ShadowResolution = UnityEngine.ShadowResolution;
+
+namespace UnityEditor.Rendering.Universal
```

Comment:
More of a general question; should all the converter things get it's own namespace?

Reply:
Added it to the new Converters namespace